fast-vm

```diff
@@ -966,9 +982,38 @@ case "$1" in
 				pmsg $P_ERROR "$3 not executable\n" $vm_number
 			fi
 		else
-			ssh -o GlobalKnownHostsFile=/dev/null -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "root@192.168.$SUBNET_NUMBER.$vm_number"
+			use_sshkey_if_exists
+			ssh $FASTVM_USE_SSHKEY -o GlobalKnownHostsFile=/dev/null -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "root@192.168.$SUBNET_NUMBER.$vm_number"
 		fi
 		;;
+	scp|keydist)
```

Comment:
I don't like having 2 actions together just to later use `if` to distinguish between them. This way we are loosing ability to do a proper argument check for SCP - where we can check if the `$3` (`sourcepath`) is a real file or not. I would really prefer to have these actions separately.